entity-management-common/src/main/java/eu/europeana/entitymanagement/common/config/EntityManagementConfiguration.java

```diff
@@ -49,6 +49,15 @@ public class EntityManagementConfiguration {
   @Value("${entitymanagement.solr.indexing.url}")
   private String indexingSolrUrl;
 
+  @Value("${entitymanagement.solr.indexing.zookeeper.url:}")
```

Comment:
must be added to entitymanagement properties with description, but commented out